campaigns/002-iw/run.py

```diff
@@ -0,0 +1,76 @@
+
+import timetracker
+
+
+def main(x):
+
+    print('Starting campaign 002-iw')
```

Comment:
As campaign output, the used print statement can be better differentiated compared to the applet output.